walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,101 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+        /*
```

Comment:
Thanks for the detailed comments Rylan! I would slightly change this comment so it reflects standard [JavaDoc practices](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html). In this case you would also move the comment right before the method declaration (i.e. line 23).